homework/01LessonOne.js

```diff
@@ -203,12 +204,12 @@ var stringFourStrictNotEqualsNumberFour;
 // What’s the difference between == and ===?
 //
 // Your Answer Goes Here:
-//
+//== is truthy (abstract) and === is an exact truth.
```

Comment:
A comparison of, but not a strict comparison which allows for room in how the logic is evaluated. @zacacollier